lib/potassium/assets/app/frontend/api/index.ts

```diff
@@ -16,6 +17,7 @@ const api = axios.create({
     'Accept': 'application/json',
     'X-CSRF-Token': csrfToken(),
   },
+  paramsSerializer: (params) => serializeParams(params),
```

Comment:
I think this could be simplified to `paramsSerializer: serializerParams` since it's passing the same argument. Or `serializerParams` could be renamed to `paramsSerializer` to use the shorthand.

Reply:
Nice, I'll go with the shorthand 👌